h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
```

Comment:
That is needed so that it will fill in the area when the card contents do not push it to the full width. So if you have a card with only two words making up the annotation, it will leave the card as a small box.